src/openforms/formio/utils.py

```diff
@@ -159,6 +159,9 @@ def get_readable_path_from_configuration_path(
 def is_layout_component(component: Component) -> bool:
     # Adapted from isLayoutComponent util function in Formio
     # https://github.com/formio/formio.js/blob/4.13.x/src/utils/formUtils.js#L25
+    if component["type"] == "editgrid":
+        return False
```

Comment:
Can you add a FIXME that this isn't the neatest way to handle this? :grimacing:

Reply:
added it